### ISSUE TITLE
Programmatically close

### ISSUE THREAD

**Issue by wsnnn** [adeptoas/sweet-modal#9]
Hello!
How can I close the already opened sweet-modal window programmatically from another sweet-modal window?
I made a variable `sm = $.sweetModal()` and the function `OnOpen` I call another sweetModal window.  In it in the function `OnClose` call` sm.close()`, the parent window is closed, but will remain overlay, which blocks html.

Maybe there is some way to close all the windows?

```javascript
var sm = $.sweetModal({
            //parent
            onOpen: function () {
                    //Some ajax actions, callback open a window
                    $.sweetModal({
                     //child
                        onClose: function () {
                            //close parent
                            sm.close();
                        }
                    });
            }
        });
```

**Comment by guillaume23** [comment]
I've the same problem, so i've tested this and it works:
`$(".sweet-modal-close-link").click();`
Thanks

**Comment by bluefirex** [comment]
@guillaume23 While this technically works it is only considered a workaround. I'm investigating for a real solution.

**Comment by bluefirex** [comment]
Fixed in edc7e512ea16a354e32034d25eadf22052ca25b4, released in [v1.3.3](https://github.com/adeptoas/sweet-modal/releases/tag/v1.3.3)
